Create an overview of the installation procedure (#4120)
theforeman/foreman-documentation: guides/common/modules/con_overview-of-installation.adoc

:_mod-docs-content-type: CONCEPT

[id="overview-of-the-installation-process"]
= Overview of the installation process

> The {Project} installation process can be broken down into three main phases:

Reviewer: ```suggestion
The {Project} installation process consists of the following main phases:
```

* Planning
* Server preparation
* Installation and configuration

.Planning
Make sure that your infrastructure meets the prerequisites for installation:

* Ensure your hardware and software meet the minimum requirements for CPU, RAM, and storage space.

> * Plan your network setup, including DNS resolution and port/firewall configurations.

Reviewer 2: "configuration" (singular)

* For large deployments (e.g., more than 5,000 hosts), select a predefined performance profile to optimize your system.

.Server preparation
Configure the operating system and the repositoties for installating {Project}:

> * Install {EL} 9 on a freshly installed system.

Reviewer 3: If we're rendering the overview for all builds, this needs to be changed for `foreman-deb` to `Debian 12 or Ubuntu 22.04` or something.

* Verify that the hostname of your {ProjectServer} resolves correctly.
* Configure your system to access the necessary repositories, which contain the {Project} packages.

.Installation and configuration
Install {Project} packages,  execute the `{foreman-installer}`, and perform any additional configuration

* Install the {Project} package and all its dependencies.
* Use the `{foreman-installer}` command to perform the initial server setup.

> * Import your subscription manifest.

Reviewer 3: If we're rendering the overview for all builds, this needs to be hidden for `foreman-{el,deb}`
